src/lib/metrics.ts

```diff
@@ -117,15 +117,29 @@ export default class MetricsMonitor {
             help: 'Number of times a feature flag has been used',
             labelNames: ['toggle', 'active', 'appName'],
         });
-        const featureFlagsTotal = createGauge({
+
+        // schedule and execute immediately
+        await dbMetrics.registerGaugeDbMetric({
```

Comment:
These tasks need to execute immediately because the tests expect that. I'd argue that some of these can be delayed, but maybe they are important because of other components depending on these. So I'm delaying this decision until we migrated all gauges to dbMetrics